### PR TITLE
feat: migration UI -  Review & Congrats page

### LINKED ISSUE #7561: Migration UI - 2 - Review
part of a bigger issue:
- #7552 

# Review
- you will get there by clicking any button with label "migrate...." on the cards 
- the UI divides here to migration of the mentioned collection or only the items inside. 

## Migrate collection
image

- top part review - sign (same as in the admin panel) 
- contextual message ( connected with)
- then there is migrate collection 
- there is picture of the collection, name and message that the name will stay the same.
- divider, and now for the harder part. 
- there is row that shows "items available for migration" 
- little message: 
   - >Please note that at this stage, only items you own can be migrated. For other items, you will need to pre-sign them to allow others to follow your migration initiative.
- If I understand correctly, you cannot migrate items you don't own. Therefore, you pre-sign for others and give them access to follow your migration once they decide to do so. 
- is that correct @vikiival ? 
- then there is route explained
image

- divider and then costs, those are also similar to the #7241 
- collapsable fee breakdown, divided into network fee and then existential deposit, and then kodadot fee (if there will be any..) 
- then a row of estimated total fees. 

### bottom part 
- here is acknowledgment before you proceed, its irreversible and you should know it. 
- disabled button till you don't check the checkbox 

## Migrate items
image
- missing available items and the info part. 
- only 2 items max are shown and then there is text of how many items are left. 
- >+ 10 more items from the same collection

### LINKED ISSUE #7563: Migration UI - 4 - Congratulations screen
part of a bigger issue:
- #7552 

# Congratulations 
- not only you have made it to the end of this issue, but the user successfully signed all the transactions and migrated. 

image

- graphics, copy, and buttons to view the collection and to share on Twitter. Copy for the tweet can be done ad hoc.

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Fix #7561
- [x] Fix #7563

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)

### <samp>🤖 Generated by Copilot at 3bc0b8f</samp>

This pull request enhances the NFT migration feature by showing and validating the fees and balances for cross-chain transfers, and by refactoring some code for better quality and maintainability. It also updates the translation key for `estimatedFees` and formats the `./locales/en.json` file.

### <samp>🤖 Generated by Copilot at 3bc0b8f</samp>

> _Sing, O Muse, of the skillful coder who devised_
> _A wondrous feature for the migration of NFTs_
> _Across the vast and varied chains of cryptic lands_
> _Where fees and balances shift like sands._

### CODE REVIEW

File: pages/migrate/congrats.vue [hunk 1]
```diff
@@ -77,4 +77,14 @@ const collectionName = computed(
...
+const collectionPage = computed(
+  () => `/${urlPrefix.value}/collection/${collectionId}`,
+)
+
+const openShareUrl = () => {
+  const shareUrl = 'https://x.com/intent/tweet?text='
+  const currentUrl = `${URLS.koda.baseUrl}${collectionPage.value}`
+
+  window.open(`${shareUrl}${encodeURIComponent(currentUrl)}`, '_blank')
+}
```
Comment (roiLeo) [inline pages/migrate/congrats.vue:89]: could this be a computed `href` instead?

File: components/migrate/steps/Review.vue [hunk 2]
```diff
@@ -213,4 +302,8 @@ const collection = computed(() =>
 .btn-submit {
   height: 3.5rem;
 }
+
+.capitalize {
```
Comment (roiLeo) [inline components/migrate/steps/Review.vue:306]: `is-capitalized`

**Review by roiLeo — APPROVED** [review]
otherwise lgtm ✅

**Comment by prury** [comment]
waiting for netlify deploy

**Comment by roiLeo** [comment]
> waiting for netlify deploy

looney-tunes-im-waiting

**Comment by prury** [comment]
So, i have this collection that has 6 items, but 3 items are not owned by me: 
https://deploy-preview-8006--koda-canary.netlify.app/rmrk/gallery/18655527-7EA1DCF47E98A25067-ANIME90-ANIME_GIRL_90S_3-0000000000000003
image

shouldn't it show burn 3 items and migrate 3 items? 

another question, does "carbon less" and "i am helping cover costs" on settings affects this process? if so, should appear on fees i suppose

**Comment by preschian** [comment]
> So, i have this collection that has 6 items, but 3 items are not owned by me:

updated


> another question, does "carbon less" and "i am helping cover costs" on settings affects this process? if so, should appear on fees i suppose

currently, I only put `I am helping cover costs`, does `carbon less NFT` option setting affect this migration cost also? @vikiival

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
pay 80 usd
let's see it live!

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $80 @ 5.59 USD/DOT ~ 14.311 $DOT
🧗 [1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p](https://polkadot.subscan.io/account/1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)
🔗 [0xa4d620809089178256e3556c1a71840c67e772cd8a1509d5fb64cc5544281dd0](https://polkadot.subscan.io/extrinsic/0xa4d620809089178256e3556c1a71840c67e772cd8a1509d5fb64cc5544281dd0)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**